domains/kwoami.json

```
@@ -0,0 +1,13 @@
+{
+    "description": "kwoami.netlify.app",
+    "repo": "https://github.com/kwoami/register/",
+    "owner": {
+        "username": "kwoami",
+        "email": "khoaquach2010@gmail.com",
```

Remove the comma (`,`)
```suggestion
        "email": "khoaquach2010@gmail.com"
```